build-ps/percona-server-8.0_builder.sh

```diff
@@ -468,19 +463,13 @@ install_deps() {
         else
             apt-get -y install libzstd1-dev
         fi
+        if [[ ${DIST} == 'focal' ]] || [[ ${DIST} == 'hirsute' ]] || [[ ${DIST} == 'bullseye' ]]; then
+            apt-get -y install libgflags-dev util-linux
```

Comment:
`util-linux` is not actually needed here. It was used only on Jenkins to get `blkzone` utility to get summary of `/dev/nullbX` devices.

Reply:
fixed